draft-ietf-masque-h3-datagram.md

```diff
@@ -570,6 +570,20 @@ all cases, the maximum permitted value of the H3_DATAGRAM SETTINGS parameter is
 1.
 
 
+## Note About Draft Versions
+
+\[\[RFC editor: please remove this section before publication.]]
+
+Some revisions of this draft specification use a different value for the
+H3_DATAGRAM Settings Parameter. This allows new draft revisions to make
+incompatible changes. Implementations MAY support multiple draft versions at
+the same time. In that scenario, endpoints MUST send multiple values for
+H3_DATAGRAM. Once an endpoint has sent and received SETTINGS, it MUST compute
```

Comment:
this is a _little_ lose. I'm likely to write an implementation that supports both but would provide an option to let people run a client or server instance that supports a) supports multiple version b) only one version. This is important if I want to force testing of a newer version without falling back to an older one. 

For a suggested rewording:
```suggestion
incompatible changes. Multiple draft versions MAY be supported by either
endpoint in a connection. Such endpoints must MUST send multiple values for
H3_DATAGRAM. Once an endpoint has sent and received SETTINGS, it MUST compute
```

Reply:
Merged without the "must MUST"